packages/dataviews/src/style.scss

```diff
@@ -373,7 +377,23 @@
 	padding: 0 $grid-unit-40;
 }
 
+.dataviews-table-selection-checkbox label {
```

Comment:
There was [a PR](https://github.com/WordPress/gutenberg/pull/57651) to do some consolidation to CSS naming. The current state is far from perfect, but could we follow the existing pattern in this one? It'd be:

```css
.dataviews-view-table-selection-checkbox label {
```

Reply:
Fixed 👍